tck/features/expressions/literals/Literals7.feature

```diff
@@ -177,7 +177,7 @@ Feature: Literals7 - List
     Given any graph
     When executing query:
       """
-      RETURN [1, -2, 00, 71034856] AS literal
+      RETURN [1, -2, 0o0, 71034856] AS literal
```

Comment:
Given the name of this test, I wonder if the list contained an octal number by mistake/typo and it should just be 0?

Reply:
Yeah I'm guessing that someone forgot that leading 0 used to mean octal, and thought `00 = 0`. However, after the deprecation cycle, `00` is actually treated as an integer I'd assume? So maybe this test can be left unchanged?

Reply:
Not necessarily. It demonstrates that the different integer literal formats can be mixed in a list. It may have been more obvious with a non-zero hex number like 0xA4C (something with a letter in it). If you want change it that respect, do not hold yourself back, it would be a win.

Reply:
That is a good question. We haven't removed this in Neo4j yet. But I would have assumed that we will throw an error for all cases that used to be deprecated incl. 00